specifyweb/frontend/js_src/lib/components/WbPlanView/Mapper.tsx

```diff
@@ -650,6 +651,15 @@ export function Mapper(props: {
                       line: line + 1,
                     });
                 }}
+                onRemoveLine={handleDeleteLine.bind(undefined, line)}
```

Comment:
callback uses word `remove`, but function uses `delete` - could you make it consistent?

Reply:
though, is ther even a reason for handleDeleteLine to exist? can't you just inline it's definition in here, or is it used in multiple places?